.gitignore

```diff
@@ -1,3 +1,36 @@
-assets/node_modules
-web/build
-assets/package-lock.json
+# OS Cruft
+.DS_Store
+
+# Package Managers
+vendor
+node_modules
+
+# Craft CMS
+config/license.key
+web/cpresources/
```

Comment:
Trailing slash. ☹️

Reply:
.gitignore has a different [usage](https://github.com/git/git/blob/1a58bad0144dedd59eeef29b7ba7791aed19106f/Documentation/gitignore.txt#L92-L113) for with and without trailing slash patterns so not modified it intentionally.

Reply:
> Let’s fix that inconsistency now.

Added trailing slashes to all the folder names.